### PR TITLE
Task about

### PR DESCRIPTION
**Pull request by daniellaskinner**
please can you review my code for about me section and media queries! thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,30 +2,53 @@
 <html lang="en">
 <head>
   <title>Daniella Skinner's Portfolio</title>
+  <meta name="viewport" content="width=device-width">
   <link rel="stylesheet" type="text/css" href="portfolioStyles.css">
+  <link rel="stylesheet" type="text/css" href="normalize.css">
```
Comment (robings) [inline index.html:6]: I would suggest putting the link to normalize.css before  your stylesheet. That way it wont override your styles.

File: index.html [hunk 2]
```diff
@@ -2,30 +2,53 @@
...
 		</div>
 	</nav>
 	
-
-	<!-- <div class="about">
-		<h3>About Me</h2>
-		<img src="portfolioImages/nelly.jpg">
+	<div class="about">
+		<h3>Full stack web developer in training</h3>
+		<img class="photo" src="portfolioImages/nelly.jpg">
```
Comment (robings) [inline index.html]: add an alt tag to the image in line 26

**Review by robings — CHANGES_REQUESTED** [review]
Great! Only a couple of comments.

**Comment by daniellaskinner** [comment]
made my changes, please approve PR!

**Review by robings — APPROVED** [review]
Approved

**Pull request merged**